galasa-parent/dev.galasa.framework.api.authentication/src/test/java/dev/galasa/framework/api/authentication/internal/routes/AuthTokensRouteTest.java

```diff
@@ -889,4 +888,176 @@ public void testAuthPostRequestWithTokenErrorResponseReturnsServerError() throws
         checkErrorStructure(outStream.toString(), 5055, "GAL5055E", "Failed to get a JWT and a refresh token from the Galasa Dex server", "The Dex server did not respond with a JWT and refresh token");
     }
 
+
+
+    @Test
+    public void testRecordingUserJustLoggedInEventWhenUserAlreadyExistButClientDoesNotShouldUpdateExistingUserRecordClientLoginTime() throws Exception {
+
+        // Given...
+        JsonObject dummyErrorJson = new JsonObject();
+        dummyErrorJson.addProperty("error", "oh no, something went wrong!");
```

Comment:
Is this error necessary for the test? Looks duplicated in the tests below as well